x/gov/types/v1/params.go

```diff
@@ -23,6 +23,7 @@ var (
 	DefaultMinExpeditedDepositTokens    = DefaultMinDepositTokens.Mul(sdkmath.NewInt(DefaultMinExpeditedDepositTokensRatio))
 	DefaultQuorum                       = sdkmath.LegacyNewDecWithPrec(334, 3)
 	DefaultThreshold                    = sdkmath.LegacyNewDecWithPrec(5, 1)
+	DefaultExpeditedQuorum              = sdkmath.LegacyNewDecWithPrec(334, 3)
```

Comment:
thoughts on leaving the expedited quorum same as default quorum?
I think setting a default expedited quorum value that is higher than 0.334 is more precise to reduce risks. Thereby enhancing the acceptance of decisions in the governance system.
cc: @alexanderbez @julienrbrt @tac0turtle

Reply:
Yes, I think it should be higher than normal quorum. I assume this is what you're proposing?

Reply:
I agree, our default should be higher, but in the migration I think we should use the same current value to not be disruptive.

Reply:
Agreed